### PR TITLE
Cleanups on the way to moving routing out of gossipd

### PR DESCRIPTION
**Pull request by rustyrussell**
Changelog-None

### CODE REVIEW

**Review by niftynei — COMMENTED** [review]
ACK https://github.com/ElementsProject/lightning/pull/4546/commits/a3e13361d7b01523d30f5acd9c9e9283281c6694

File: common/gossmap.h [hunk 1]
```diff
@@ -111,6 +111,11 @@ static inline bool gossmap_chan_set(const struct gossmap_chan *chan, int dir)
 	return chan->half[dir].htlc_max != 0;
 }
 
+/* Return capacity if it's known (fails for private channels) */
```
[the remark this answers is not in the document]
Comment (rustyrussell) [reply common/gossmap.h]: Fixed!

**Comment by rustyrussell** [comment]
Push to fix comment and shellcheck.

**Comment by vincenzopalazzo** [comment]
ack https://github.com/ElementsProject/lightning/pull/4546/commits/4a0f73e9991faa5f292155cfc956d6eb798fe9a6

**Comment by rustyrussell** [comment]
Another trivial rebase to fix up mocks.

**Pull request merged**